test/test-server/org_handlers.go

```diff
@@ -108,9 +109,9 @@ func handleOrgOrganizations(t *testing.T) http.HandlerFunc {
 		switch r.Method {
 		case http.MethodGet:
 			organizationList := &orgv2.OrgV2OrganizationList{Data: []orgv2.OrgV2Organization{
-				{Id: orgv2.PtrString("abc-123"), DisplayName: orgv2.PtrString("org1")},
-				{Id: orgv2.PtrString("abc-456"), DisplayName: orgv2.PtrString("org2")},
-				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3")},
+				{Id: orgv2.PtrString("abc-123"), DisplayName: orgv2.PtrString("org1"), JitEnabled: orgv2.PtrBool(true)},
+				{Id: orgv2.PtrString("abc-456"), DisplayName: orgv2.PtrString("org2"), JitEnabled: orgv2.PtrBool(true)},
+				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3"), JitEnabled: orgv2.PtrBool(true)},
```

Comment:
Let's make one of these `false`, just for some variation.
```suggestion
				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3"), JitEnabled: orgv2.PtrBool(false)},
```